datafusion/core/src/datasource/file_format/parquet.rs

```diff
@@ -1439,6 +1441,57 @@ mod tests {
         Ok(())
     }
 
+    #[tokio::test]
+    async fn test_statistics_from_parquet_metadata_dictionary() -> Result<()> {
+        // Data for column c_dic: ["a", "b", "c", "d"]
+        let values = StringArray::from_iter_values(["a", "b", "c", "d"]);
+        let keys = Int32Array::from_iter_values([0, 0, 1, 2]);
```

Comment:
```suggestion
        let keys = Int32Array::from_iter_values([0, 1, 2, 3]);
```
My bad that in my original test example I gave these confusing dictionary keys 😅 I think `0, 1, 2, 3` makes more sense since the dictionary values are `a, b, c, d` -- which in different values